midas/input_parser.py

```diff
@@ -475,6 +492,7 @@ def parse_input_data(self):
         
         self.population_size = yaml_line_reader(info, 'population_size', 1)
         self.num_generations = yaml_line_reader(info, 'number_of_generations', 1)
+        self.batch_size = yaml_line_reader(info, 'batch_size', 1)
```

Comment:
what is batch_size used for? how does it differ from population_size?

Reply:
batch size is essentially the same as population size, where it represents the number of individuals created and evaluated per generation. However, I intend to use this in order to have a population size number of initial random points and then batch size number of individuals per generation. The reason is because the number of initial points and number of points per iteration can affect the optimization, where sometimes having a relatively small or large batch size can help it focus more on exploring a wider range of solutions or in finding a more precise solution.